changelogs/unreleased/gh-8146-space-and-index-names-in-iproto-requests.md

```diff
@@ -0,0 +1,10 @@
+## feature/box
+
+* Added support for accepting IPROTO requests with a space or index name instead
+  of an identifier (gh-8146).
+
+## feature/net.box
+
+* Added support for sending CRUD requests from connection objects with disabled
+  schema fetching by manually specifying a space or index name or an identifier
+  (gh-8146).
```

Comment:
> I think it's better to merge these two changelog entries in one, because it's a single IPROTO feature. The built-in net.box connector must be up-to-date with the current IPROTO implementation anyway.

Done.

> I'd also put the entry under the `feature/core` category.

But IPROTO is part of box.

Reply:
> But IPROTO is part of box.

I think that the box is a part of the core. Actually, there's no section "box" in the known section list:

https://github.com/tarantool/tarantool/blob/a27d958d060ae2725ed8df6c1a4cc2a51d776233/tools/gen-release-notes#L14-L17

Unfortunately, we don't have an automated tool to check the section name nor do we have a clear guideline about how to choose a section name so we use "box" now and then and nobody cares and you can continue using "box" if you like. (I filed a ticket to enable section name checking some time ago - see #6996.)

Reply:
AFAIC, core is stuff like fibers and other primitives build on — `box` is database stuff.